az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/AbstractHadoopJavaProcessJob.java

```diff
@@ -62,10 +62,6 @@ public AbstractHadoopJavaProcessJob(String jobid, Props sysProps, Props jobProps
   public void run() throws Exception {
     try {
       super.run();
-    } catch (Throwable t) {
```

Comment:
Since this is to deal with an NPE why not catch that specifically here instead of catching throwable?